src/Language/PureScript/Parser/Declarations.hs

```diff
@@ -231,6 +237,7 @@ parseLocalDeclaration :: TokenParser Declaration
 parseLocalDeclaration = positioned (P.choice
                    [ parseTypeDeclaration
                    , parseValueDeclaration
+                   , parseBoundValueDeclaration
```

Comment:
I think it would be better to always try to parse many binders here, and then turn the single `VarBinder` case into a `ValueDeclaration`. That way we can avoid backtracking in the parser.